src/main/java/org/jitsi/jigasi/stats/Statistics.java

```diff
@@ -297,6 +357,10 @@ public static synchronized void sendJSON(
         stats.put(TOTAL_CALLS_JVB_NO_MEDIA, totalCallsJvbNoMedia.get());
         stats.put(TOTAL_CALLS_NO_HEARTBEAT, totalCallsWithNoHeartBeatResponse.get());
 
+        stats.put(TOTAL_TRANSCRIBER_STARTED, totalTrasnscriberStarted.get());
+        stats.put(TOTAL_TRANSCRIBER_STOPPED, totalTrasnscriberStopped.get());
+        stats.put(TOTAL_TRANSCRIBER_FAILED, totalTrasnscriberFailed.get());
+
```

Comment:
totalTrasnscriberRequests and total15sIntervals don't get exposed here?

Reply:
Done.